docs/examples.qmd

```diff
@@ -24,17 +24,17 @@ import matplotlib.pyplot as plt
 import seaborn as sns
 
 # Just import maidr package #<<
-import maidr #<<
+import maidr  #<<
 
 
 # Load the penguins dataset
 penguins = sns.load_dataset("penguins")
 
 # Create a bar plot showing the average body mass of penguins by species
-plt.figure(figsize=(10, 6))
+plt.figure(figsize=(6, 6))
```

Comment:
What if you use the cell-level fig size adjustment in the beginning of this cell instead of the plot-level adjustment? e.g.,


```suggestion
#| fig-width: 6
#| fig-height: 6

```